sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -347,6 +347,7 @@ def query_items(
         max_item_count=None,  # type: Optional[int]
         enable_scan_in_query=None,  # type: Optional[bool]
         populate_query_metrics=None,  # type: Optional[bool] # pylint:disable=docstring-missing-param
+        populate_index_metrics=None, # type: Optional[bool]
```

Comment:
I think we should just put this into the kwargs the same way we do for the async client - I see you're going for uniformity here but ideally we would have had all of those additional options in kwargs to begin with